platforms/jvm/language-java/src/main/java/org/gradle/external/javadoc/JavadocOptionFileOption.java

```diff
@@ -25,5 +26,6 @@
  */
 @HasInternalProtocol
 public interface JavadocOptionFileOption<T> extends OptionLessJavadocOptionFileOption<T> {
+    @ToBeKeptEagerProperty(because = "TODO: Decide what to do with that")
```

Comment:
❓ Shall we instead remove the annotation and keep the exception in the freeze file?

Reply:
Done